bpm/bonita-core/bonita-process-engine/src/main/java/org/bonitasoft/engine/api/impl/PlatformAPIImpl.java

```diff
@@ -655,9 +644,7 @@ private void createDefaultTenant(final PlatformServiceAccessor platformAccessor,
             sessionAccessor.deleteSessionId();
             sessionAccessor.setSessionInfo(session.getId(), tenantId);// necessary to create default data source
 
-            // Create default profiles
-            createDefaultProfiles(tenantServiceAccessor);
-
+            // Create default profiles: done by profile updater restart handler
```

Comment:
I finally added the creation also in platformAPI because a test was failing (on postgres) due to the asynchronism. The "after service start" of the profile restart handler is called in a separated thread after the engine start. because of the when the engine is started the first time, there is a laps of time during which profile are not here